test/micro-distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -1,4 +1,10 @@
 @Builtin_Type
 type Vector a
     from_array array = @Builtin_Method "Vector.from_array"
-    at self index = @Builtin_Method "Vector.at"
+    at self index = array_like_at self index
```

Comment:
Why is it that we need an intermediate method here? Why cannot we simply write `at self index = @Builtin_Method "Array_Like_Helpers.at`?